### ISSUE TITLE
scriptcomp fails to understand positive sign on ints

### ISSUE THREAD

**Issue by niv** [niv/neverwinter.nim#57]
```nwscript
void main()
{
  int a = -1;
  int b = +1; // ERROR: UNKNOWN STATE IN COMPILER
  int c = +a; // ERROR: UNKNOWN STATE IN COMPILER
}
```

This works fine on nwnsc.

**Comment by mtijanic** [comment]
Looks like it actually doesn't have the unary plus operator at all. Not that it matters much, but if we take the official compiler spec to be the language spec, then this was never considered and therefore more of a missing feature than a bug. Absolutely useless distinction, except that whoever takes this up should know that it needs changes at all levels and not a point fix.

**Comment by mtijanic** [comment]
Fixed by #130
